bin/ingestion.js

```diff
@@ -164,11 +145,11 @@ function setupZkLocationNode(zkClient, location, done) {
                     });
                     return done(e);
                 }
-                if (d[RESUME_NODE]) {
-                    return checkAndApplyScheduleResume(zkClient, d, location,
-                        done);
+                if (locationData[RESUME_NODE]) {
+                    return checkAndApplyScheduleResume(zkClient, locationData,
+                        location, done);
                 }
-                if (data.paused === true) {
+                if (data.paused) {
```

Comment:
should this data be locationData?

Reply:
Well `data` is converted to string, parsed as json again and assigned to `locationData`. Not sure why this was done, but going to keep and for sanity, changing it to `locationData`.

Reply:
Seems like zookeeper ( according to docs ) passes data as a raw Buffer, but that it is probably JSON data so that explains the toString + JSON.parse. Meaning data.paused would never evaluate to anything as a byte buffer has no paused attribute.

Reply:
No, I meant in the original implementation, it was `data.paused`.

Reply:
oh... that doesnt sound good. Are there specs or docs related to what the data comes back as? Data is defined by zookeeper as a Buffer, but what is actually in the buffer is defined somewhere else. Hard to say if this is correct without it. It could be isPaused or wasPaused for all I know.

Reply:
well I corrected it.